assets/src/dashboard/components/infiniteScroller/index.js

```diff
@@ -77,22 +81,36 @@ const loadReducer = (state, action) => {
 
 const InfiniteScroller = ({
   allDataLoadedMessage = __('No More Stories', 'web-stories'),
+  allDataLoadedAriaMessage = __('All stories are loaded', 'web-stories'),
   onLoadMore,
   canLoadMore,
   isLoading,
+  loadingAriaMessage = __('Loading more stories', 'web-stories'),
   loadingMessage = __('Loading…', 'web-stories'),
 }) => {
   const loadingRef = useRef(null);
   const onLoadMoreRef = useRef(onLoadMore);
   onLoadMoreRef.current = onLoadMore;
 
   const [loadState, dispatch] = useReducer(loadReducer, STATE.loadable);
+  const [loadingAlert, setLoadingAlert] = useState(null);
```

Comment:
Nit, instead of using state, can loadingAlert be defined with `useMemo`?  Something like this:
```
const loadingAlert = useMemo(() => {
  if (loadState === STATE.loading_internal) {
    return loadingAriaMessage;
  } else if (!canLoadMore) {
    return allDataLoadedAriaMessage;
  } 
  
  return null;
}, [loadingAriaMessage, allDataLoadedAriaMessage, canLoadMore, loadState]);
```

This way the loadingAlert will have only one function to determine what message we want to alert.

Reply:
I like the idea of using memo to compute something transient rather than saving them in `useState`.

Reply:
great idea! updating now, thank you!